elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -485,31 +392,24 @@ export abstract class BaseTransaction {
 		}
 		// tslint:disable-next-line no-magic-numbers
 		const timeNow = Math.floor(date.getTime() / 1000);
-		const timeOut =
-			this._multisignatureStatus === MultisignatureStatus.PENDING ||
-			this._multisignatureStatus === MultisignatureStatus.READY
-				? UNCONFIRMED_MULTISIG_TRANSACTION_TIMEOUT
-				: UNCONFIRMED_TRANSACTION_TIMEOUT;
 		const timeElapsed =
 			// tslint:disable-next-line no-magic-numbers
 			timeNow - Math.floor(this.receivedAt.getTime() / 1000);
 
-		return timeElapsed > timeOut;
+		return timeElapsed > UNCONFIRMED_TRANSACTION_TIMEOUT;
```

Comment:
What's this for? Is it related to Transaction Pool changes?

Reply:
yes, we have 3-hour timeout for all types of transactions, before it was calculating for multi-sig as well, so removed those specific calculations.